taxed-fungible-asset/move/Move.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "taxed_fa"
+version = "0.0.0"
+
+[dev-addresses]
+deployer = "0xcafe"
```

Comment:
probably only my habit, but i usually name the address of the package something like `launchpad_addr` or `tax_fa_addr` in this case

Reply:
found deployer to be a better name for this since we're using it for permission checks (onlyDeployer type of modifier) but open to changing for consistency across the repo

Reply:
sounds good! no strong opinion from me